patches/minecraft/net/minecraft/enchantment/Enchantment.java.patch

```diff
@@ -18,7 +34,7 @@
     }
  
     public void func_151368_a(LivingEntity p_151368_1_, Entity p_151368_2_, int p_151368_3_) {
-@@ -139,6 +139,24 @@
+@@ -139,6 +148,24 @@
```

Comment:
What's up with these hunks being changed randomly?
I don't see a reason for these to be changed, from looking at the files

Reply:
iirc it is just because that represents what line it is starting on so because there are more lines being injected above that means that anything being added lower down gets added *later* line number wise.

Reply:
I don't see any other patches in these files, these should not be changed?
Else every single line would have one of these..

Reply:
Same with the first one in this file. You don't change anything above it, yet it is still changed.

Reply:
https://github.com/MinecraftForge/MinecraftForge/blob/1.16.x/patches/minecraft/net/minecraft/enchantment/Enchantment.java.patch#L21-L42

Reply:
Okay. that one was me being blind, my bad.
Now explain this: https://github.com/MinecraftForge/MinecraftForge/pull/7379/files#diff-9b95d3b9cf5c13f0915da0244317d899R3

Reply:
There are 9  lines added to the file and the patch lines below are shifted by 9, seems pretty sane to me

Reply:
Because a line is being added here curle? https://github.com/MinecraftForge/MinecraftForge/pull/7379/files#diff-9b95d3b9cf5c13f0915da0244317d899R9

Reply:
The first one. There is nothing being added above it, and the line being added inside it shouldn't affect the position.
Whatever, i'll just chalk it down to Git being weird.
I redact my first statement :P